pkgs/os-specific/linux/fswebcam/default.nix

```
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "fswebcam";
-  version = "2020-07-25";
+  version = "20200725";
 
   src = fetchurl {
     url = "https://www.sanslogic.co.uk/fswebcam/files/fswebcam-${lib.replaceStrings ["."] [""] version}.tar.gz";
```

To keep the current version (`2020-07-25`):

```suggestion
    url = "https://www.sanslogic.co.uk/fswebcam/files/fswebcam-${lib.replaceStrings ["-"] [""] version}.tar.gz";
```

To keep the changed version (`20200725`):

```suggestion
    url = "https://www.sanslogic.co.uk/fswebcam/files/fswebcam-${version}.tar.gz";
```

---

Okay I guess `20200725` is the current one according to repology.